CHANGELOG.md

```diff
@@ -4,11 +4,13 @@
 
 ### Added
 
+- Added skip-duplicate-runs #423
 - Bump: terraform 0.13.4 ~> 0.13.5 #445
 - Bump: consul 1.8.4 ~> 1.8.5 #445
 - Bump: vault 1.5.4 ~> 1.6.0 #445
 - Bump: nomad 0.12.5 ~> 0.12.8 #445
 - Bump: packer 1.6.4 ~> 1.6.5 #445
+-test
```

Comment:
```suggestion
```

Reply:
Thanks, this was just so I could get that one commit in to test :smile: